### ISSUE TITLE
Big blue box on top of user profile details

### ISSUE THREAD

**Issue by rfay** [warmshowers/Warmshowers.org#669]
On my build there's a big blue box on top of the user details, picture:

![nancy_lewis_and_randy_fay___warmshowers_org](https://cloud.githubusercontent.com/assets/112444/9482004/9c9e84bc-4b4c-11e5-9f61-da15a010fcb7.png)

**Comment by rfay** [comment]
Seems like the grid is in the wrong place or something.

Editing line 99 of _responsive.scss can fix this, but I suspect it's the wrong solution:

```
diff --git a/docroot/sites/all/themes/warmshowers_zen/sass/layouts/_responsive.scss b/docroot/sites/all/themes/warmshowers_zen/sass/layouts/_responsive.scss
index 88f98f9..be725a0 100644
--- a/docroot/sites/all/themes/warmshowers_zen/sass/layouts/_responsive.scss
+++ b/docroot/sites/all/themes/warmshowers_zen/sass/layouts/_responsive.scss
@@ -97,7 +97,7 @@ $zen-auto-include-item-base: false;
   .main-wrapper {

     .region-highlighted {
-      @include zen-grid-item(9, 1);
+      @include zen-grid-item(28, 1);
     }

     .main {
@@ -135,4 +135,4 @@ $zen-auto-include-item-base: false;
       display: block;
     }
   }
-}
```

**Comment by rfay** [comment]
Fixed in #737 by @kalpaitch - Thanks!
